### DISCUSSION TITLE
The version number construct with 'y'

### DISCUSSION THREAD

**Discussion by IwateKyle** [solidjs/solid-start#722] [Q&A] [answered]
I have been noticing this pattern a few times recently and don't recall seeing it in the past.  
In the version label I see the progression from alpha->beta-> ...y....
what does the 'y' mean?  I have searched but there is no easy way to search for it.
ex: ```"version": "0.1.0-y.0"```

**Comment by ryansolid** [comment, marked as answer]
That was a really old version of alpha where I accidentally hit `y` which means yes for version instead of `enter` on a specific prompt of Lerna's deploy command. Most of the prompts what `y` or `n`. I think I remember it being late and I just pressed `y` a few times and there we are.

**Reply by IwateKyle** [reply]
ah!  I get it.  I feel much better now. I think another project I looked at recently had a similar 'y' in it, so I thought there was a new trend.  Fair enough.  Thanks for the response.

**Reply by IwateKyle** [reply]
Does the master branch point to the alphas and betas - or only the latest release-ish versions?  I'm showing off my lack of git knowledge here, I know.
